### ISSUE TITLE
Associate acceleration measurements to deployment, not to separate event for each

### ISSUE THREAD

**Issue by peterdesmet** [tdwg/dwc-for-biologging#6]
@sarahcd, just looked at your "Mahoney-data-DwC-A-test-2". This is the structure I could detect:

event_id | events | assoc occ | assoc event mof | assoc occ mof
--- | --- | --- | --- | ---
F53:capture1 | 1 | 1 | 0 | 0
F53:recovery1 | 1 | 1 | 0 | 0
IdCoy_P3_1:capture1 | 1 | 1 | 0 | 0
IdCoy_P3_1:recovery1 | 1 | 1 | 0 | 0
IdCoy_P3_1:capture2 | 1 | 1 | 0 | 0
IdCoy_P3_1:recovery2 | 1 | 1 | 0 | 0
F53:deployment1 | 1 | 1361 | 0 | 1361 * 4
IDCoy_P3_1:deployment1 | 1 | 10809 | 0 | 0
IDCoy_P3_1:deployment2 | 1 | 15300 | 0 | 0
F53:deployment1:xxx | 76869 | 0 | 76869 * 2 | 0

I would simplify this to:

event_id | events | assoc occ | assoc event mof | assoc occ mof
--- | --- | --- | --- | ---
F53:capture1 | 1 | 1 | 0 | 0
F53:recovery1 | 1 | 1 | 0 | 0
IdCoy_P3_1:capture1 | 1 | 1 | 0 | 0
IdCoy_P3_1:recovery1 | 1 | 1 | 0 | 0
IdCoy_P3_1:capture2 | 1 | 1 | 0 | 0
IdCoy_P3_1:recovery2 | 1 | 1 | 0 | 0
F53:deployment1 | 1 | 1361 | 76869 * 2 | 1361 * 4
IDCoy_P3_1:deployment1 | 1 | 10809 | 0 | 0
IDCoy_P3_1:deployment2 | 1 | 15300 | 0 | 0

All acceleration measurements are done in `F53:deployment1`. Just like you group all occurrences under that event (with each occ. having a more precise `eventDate`), I would group all acceleration measurements under that same event (with each meas. having a more precise `measurementDeterminedDate`). That way, the Event core is uncluttered and gives a quick overview of captures, recoveries and deployments.

Note, since acceleration measurements are measured independently from GPS positions (correct me if untrue), it is not possible to flatten the Events into the occurrences, which is an approach I like, but have to reconsider 😄 .

**Comment by sarahcd** [comment]
Thanks @peterdesmet! This update will be made in Mahoney use case V3, which I'm working on now. (Although I'll be grateful for a check before we close this.) As you suspect, the acceleration measurements because they were not collected on the same schedule as GPS fixes. This will be pretty common for deployments of multi-sensor tags (and tags with no location sensors) and is one reason I chose this sample dataset. Thanks for pointing out measurementDeterminedDate. I agree it doesn't seem ideal to spread the measurement timestamps across two tables and terms, I don't have a better suggestion, but maybe something to discuss further.

One last related question: Can I use BasisOfRecord in the event or FOM tables? Currently in my example I don't define the measurements without occurrences as MachineObservation.

**Comment by peterdesmet** [comment]
> Can I use BasisOfRecord in the event or FOM tables? Currently in my example I don't define the measurements without occurrences as MachineObservation.

No, you can't. There is no way to differentiate between machine or human measurements in MOF. I think you will have to rely on the definition in `measurementTypeID`

`
